doc/api-extensions.md

```diff
@@ -1901,6 +1901,10 @@ client authentication.
 
 Adds ability to copy image to a project different from the source.
 
+## `images_all_projects`
+
+This adds support for displaying images from all projects.
```

Comment:
I'd probably be a bit more specific with something like:

```
This adds support for listing images across all projects through the `all-projects` parameter on `GET /1.0/images`.
```

Reply:
Done